submission-panels.md

```diff
@@ -0,0 +1,80 @@
+# Submission-panels Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/submission-definitions**   
+
+Provide access to the configured submission definitions. It returns the list of existent submission-definitions.
+
+Example: to be provided
+
+## Single Submission-panels 
+**/api/configuration/submission-definitions/<:definition-name>**
+
+Provide detailed information about a specific submission-definition. The JSON response document is as follow
+```json
+{
+  "name": "traditional",
+  "panels": [
+  {
+  	id: "id-of-the-input-form-page",
+  	header: "First page",
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null,
+  	_links: {
+  		"configuration" : "<dspace-url>/configuration/inputforms/<:id-of-the-input-form-page>" 
+  	}
+  },
+  {
+    id: "id-of-the-input-form-page",
+  	header: "Second page",
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null,
+  	_links: {
+  		"configuration" : "<dspace-url>/configuration/inputforms/<:id-of-the-input-form-page>" 
+  	}
+  },
+  {
+    id: "id-of-the-upload-panel",
+  	header: "Files and access condition",
+  	mandatory: true,
+  	type: "uploadWithEmbargo",
+  	scope: null
+  },
+  {
+  	id: "id-of-the-deposit-license-panel",
+  	header: "Deposit license",
+  	mandatory: true,
+  	type: "license",
+  	scope: null
+  },
+  {
+  	id: "id-of-the-cc-license-panel",
+  	header: "Creative Commons",
```

Comment:
Same remark here on the `configuration` link. You also need that here to retrieve the list of enabled CC licenses.